batchflow/tests/segmentation_metrics_by_pixels_test.py

```diff
@@ -0,0 +1,312 @@
+"""
+Tests for SegmentationMetricsByPixels class from batchflow.models.metrics.
+Structurally, file consists of two main classes. First one tests metric's output shape, second — its contents.
+Main test functions run for almost all metrics values, declared in metrics_dict (exceptions described below).
+Test data is pre-defined, it's shape and contents were chosen for reasons of balance between visual simplicity
+and test coverage diversity.
+"""
+# pylint: disable=import-error, no-name-in-module
+from itertools import chain
+import numpy as np
+import pytest
+
+from batchflow.models.metrics import SegmentationMetricsByPixels
+
+# Dict {metric_alias : metric_names}
+METRICS_DICT = {'tpr' : ['true_positive_rate', 'sensitivity', 'recall'],
```

Comment:
Let's add aliases for metrics into metrics classes. It will be easier to write something like 
`metrics.evaluate('fpr')` then `metrics.evaluate('false_positive_rate')`

Reply:
`METRICS_ALIASES` added to `classify.py`